test/EFCore.MySql.IntegrationTests/Tests/Models/GeneratedTypesTest.cs

```diff
@@ -12,7 +12,7 @@ namespace Pomelo.EntityFrameworkCore.MySql.IntegrationTests.Tests.Models
 {
 	public class GeneratedTypesTest
 	{
-		[SkipAppVeyorAndMariaDbFact]
+		[SkipJsonFact]
```

Comment:
`SupportedServerVersionFactAttribute` can be used here like one of the two following lines:
```c#
[SupportedServerVersionFactAttribute(ServerVersion.JsonSupportKey)]
// or
[SupportedServerVersionFactAttribute(ServerVersion.JsonSupportKey, Skip = "Version of MySQL/MariaDB does not support JSON")]
```

Reply:
Changed to `[SupportedServerVersionFact(ServerVersion.JsonSupportKey, Skip = "Version of MySQL/MariaDB does not support JSON")]`